src/Packager.php

```diff
@@ -334,6 +392,110 @@ public function createZipArchive(): bool
         return $this->createZipArchive;
     }
 
+    /**
+     * Set meta description for SCORM package
+     *
+     * @param string|null $metadataDescription
+     */
+    public function setMetadataDescription(?string $metadataDescription): void
+    {
+        if (!$metadataDescription) {
```

Comment:
You shouldn't use a negative condition if you handle both results of the condition. In this case better to use `is_null` instead of implicit check.